modules/swagger-parser-v2-converter/src/test/java/io/swagger/parser/test/V2ConverterTest.java

```diff
@@ -253,6 +255,28 @@ public void testIssue14() throws Exception {
     public void testIssue15() throws Exception {
         OpenAPI oas = getConvertedOpenAPIFromJsonFile(ISSUE_15_JSON);
         assertNotNull(oas);
+
+        Info info = oas.getInfo();
+        assertNotNull(info.getExtensions().get("x-apis-json"));
+        assertNotNull(info.getLicense().getExtensions().get("x-notes"));
+        assertNotNull(oas.getExternalDocs().getExtensions().get("x-docs-extension"));
+        assertNotNull(oas.getTags().get(0).getExtensions().get("x-tag-extension"));
+        assertNotNull(oas.getTags().get(0).getExternalDocs().getExtensions().get("x-tag-docs-extension"));
+
+        PathItem pathItem = oas.getPaths().get("/something");
+        assertNotNull(pathItem.getExtensions().get("x-path-item-extension"));
+
+        Operation get = pathItem.getGet();
+        assertNotNull(get.getExtensions().get("x-version"));
+        assertNotNull(get.getExternalDocs().getExtensions().get("x-operation-docs-extension"));
+        assertNotNull(get.getResponses().get("200").getExtensions().get("x-response-extension"));
+
+        ArraySchema schema = (ArraySchema) get.getParameters().get(0).getSchema();
+        assertNotNull(schema.getItems().getExtensions().get("x-example"));
```

Comment:
`x-example`, `x-examples`, `x-nullable` should not be moved as extensions since they are converted to their 'native' equivalent in OAS3.

Reply:
Done!